src/core_modules/capture-core/components/Pages/common/TEIRelationshipsWidget/RegisterTei/RegisterTei.container.js

```diff
@@ -14,26 +13,36 @@ export const RegisterTei = ({
     suggestedProgramId,
 }: ContainerProps) => {
     const dataEntryId = 'relationship';
-    const itemId = useSelector(({ dataEntries }) => dataEntries[dataEntryId]?.itemId);
+    // const itemId = useSelector(({ dataEntries }) => dataEntries[dataEntryId]?.itemId);
     const error = useSelector(({ newRelationshipRegisterTei }) => (newRelationshipRegisterTei.error));
     const selectedScopeId = suggestedProgramId || trackedEntityTypeId;
     const { trackedEntityName } = useScopeInfo(selectedScopeId);
-    const { buildTeiPayload } = useDataEntryReduxConverter({
+    /* const {
```

Comment:
There are some commented lines of code in this file. Can you clean them before merging? Thanks!